lib/tasks/fake_data.rake

```diff
@@ -216,11 +216,60 @@ namespace :fake_data do
     end
   end
 
+  desc 'create fake journeys'
+  task create_journeys: :environment do
+    puts 'create_journeys...'
+    Move
+        .left_outer_joins(:journeys).where(journeys: { move_id: nil })
+        .where(status: %w[completed booked requested])
+        .where.not(from_location: nil)
```

Comment:
will we ever have a `from_location` nil? Both journey and move validate against this?

Reply:
good point well made 👍